Description: lxc-clone: fix wrong handling of lxc.mount
 We can't simply search on and remove 'lxc.mount' entries, as those will
 also hit lxc.mount.entry entries.  Look for 'lxc.mount[ \t]' instead.
 This cherrypicks an upstream fix for lxc-clone, so does not need to be
 forwarded upstream.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1084089
Forwarded: no

Index: lxc/src/lxc/lxc-clone.in
===================================================================
--- lxc.orig/src/lxc/lxc-clone.in	2012-11-26 16:52:52.161010000 -0600
+++ lxc/src/lxc/lxc-clone.in	2012-11-28 10:47:07.365481892 -0600
@@ -180,7 +180,7 @@
 sed -i '/lxc.utsname/d' $lxc_path/$lxc_new/config
 echo "lxc.utsname = $hostname" >> $lxc_path/$lxc_new/config
 
-grep "lxc.mount" $lxc_path/$lxc_new/config >/dev/null 2>&1 && { sed -i '/lxc.mount/d' $lxc_path/$lxc_new/config; echo "lxc.mount = $lxc_path/$lxc_new/fstab" >> $lxc_path/$lxc_new/config; }
+grep "lxc.mount[ \t]" $lxc_path/$lxc_new/config >/dev/null 2>&1 && { sed -i '/lxc.mount[ \t]/d' $lxc_path/$lxc_new/config; echo "lxc.mount = $lxc_path/$lxc_new/fstab" >> $lxc_path/$lxc_new/config; }
 
 if [ -e  $lxc_path/$lxc_orig/fstab ];then
     cp $lxc_path/$lxc_orig/fstab $lxc_path/$lxc_new/fstab
